test/set/symmetric_difference.cpp

```diff
@@ -2,8 +2,6 @@
 // Distributed under the Boost Software License, Version 1.0.
 // (See accompanying file LICENSE.md or copy at http://boost.org/LICENSE_1_0.txt)
 
-#include <boost/hana/assert.hpp>
```

Comment:
Yeah, but you should always include what you use to avoid relying on such dependencies, no?